Description: Send ack on apply diff
 Fix ApplyUpdate() to return an empty string as per spec if the update
 is successfull (LP: #1260712)
Author: Didier Roche <didrocks@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1260712

--- system-image-2.0.3.orig/systemimage/dbus.py
+++ system-image-2.0.3/systemimage/dbus.py
@@ -233,6 +233,7 @@ class Service(Object):
     def ApplyUpdate(self):
         """Apply the update, rebooting the device."""
         GLib.timeout_add(50, self._apply_update)
+        return ""
 
     @method('com.canonical.SystemImage', out_signature='isssa{ss}')
     def Info(self):
